src/main/java/com/manywho/services/email/email/decisions/clients/EngineClient.java

```diff
@@ -34,12 +37,13 @@ public String sendResponseToEngine(EmailDecisionRequest emailRequest, String str
                     .execute();
 
             if (response.isSuccessful() == false) {
+                LOGGER.error("The response from engine is not successful: "+ response.message());
```

Comment:
Can you change this to use a parameter like `LOGGER.error("The response from engine is not successful: {}", response.message())`, otherwise each message will come up as unique, but the parameter will make it group generically